### ISSUE TITLE
Audio file

### ISSUE THREAD

**Issue by penpadstudio** [NHSLeadership/nightingale#159]
<img width="656" alt="screen shot 2017-09-08 at 11 20 39" src="https://user-images.githubusercontent.com/23170050/30207531-c5845560-9487-11e7-8660-cee38531aff2.png">
@cehwitham set priority please

**Comment by penpadstudio** [comment]
@ishanivardhan @csswizardry @cehwitham This has not been built into v1 Nightingale from what I can remember this may have been dropped for v1 launch. Anybody else remember? Also tagged you @ishanivardhan  as you were previously asking about Audio file designs.

**Comment by ishanivardhan** [comment]
Yes @Pro-Paul if and when the designs on this get finalised, would be great to see it Live.
I do have an outstanding issue wrt white background of audio on white screen in my Moodle theme repo. https://github.com/NHSLeadership/moodle-theme_nightingale/issues/83
